main/util/src/mill/util/ProcessUtil.scala

```diff
@@ -0,0 +1,15 @@
+package mill.util
+
+import mill.api.Result
+
+object ProcessUtil {
+
+  def toResult(processResult: os.CommandResult): Result[Int] = {
```

Comment:
If we make `Jvm.callSubprocess` take `check: Boolean = true`, then we can just rely on the underlying `os.call` throwing an exception and won't need to call this `ProcessUtil.toResult` everywhere